salt/grains/iscsi.py

```diff
@@ -103,7 +100,7 @@ def _windows_iqn():
     get = "iSCSINodeName"
 
     cmd_ret = salt.modules.cmdmod.run_all(
-        "{0} /namespace:{1} path {2} get {3} /format:table"
+        "{} /namespace:{1} path {2} get {3} /format:table"
```

Comment:
looks like it's here that it doesn't like, sounds like it should just be `{}`'s and not `{1}`, `{2}`, etc.